lib/internal/module.js

```diff
@@ -130,3 +121,12 @@ function addBuiltinLibsToObject(object) {
     });
   });
 }
+
+module.exports = exports = {
```

Comment:
Why? There is no point. Since you are already exposing the variable name to grab it this way `exports` is defunct.

If necessary, we can remove this in another PR entirely but idk why we'd add more here...